drivers/include/mcp23017.h

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_mcp23017
+ * @ingroup     drivers_actuators
+ * @brief       Device driver interface for the MCP23017
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the MCP23017
+ *
+ * @author      Dimitri Nahm <dimitri.nahm@haw-hamburg.de>
+ */
+
+#ifndef MCP23017_H
+#define MCP23017_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+/**
+ * @brief   Possible MCP23017 hardware addresses (wiring specific)
+ */
+enum {
+    MCP23017_ADDR_20 = 0x20, /**< I2C device address: A2=0, A1=0, A0=0 */
+    MCP23017_ADDR_21 = 0x21, /**< I2C device address: A2=0, A1=0, A0=1 */
+    MCP23017_ADDR_22 = 0x22, /**< I2C device address: A2=0, A1=1, A0=0 */
+    MCP23017_ADDR_23 = 0x23, /**< I2C device address: A2=0, A1=1, A0=1 */
+    MCP23017_ADDR_24 = 0x24, /**< I2C device address: A2=1, A1=0, A0=0 */
+    MCP23017_ADDR_25 = 0x25, /**< I2C device address: A2=1, A1=0, A0=1 */
+    MCP23017_ADDR_26 = 0x26, /**< I2C device address: A2=1, A1=1, A0=0 */
+    MCP23017_ADDR_27 = 0x27, /**< I2C device address: A2=1, A1=1, A0=1 */
+};
```

Comment:
No need to list all these addresses explicitly I'd say. Having these in a comment somewhere should suffice.